website/tutorial/testcontainers-with-podman.md

```diff
@@ -0,0 +1,223 @@
+   }
+
+   export async function disconnectRedis() {
+       if (redisClient) {
+           await redisClient.quit();
+           redisClient = undefined;
+       }
+   }
+   ```
+
+5. Create basic tests for CRUD operations.
+
+   ```index.spec.ts
+   import { afterAll, beforeAll, beforeEach, expect, test } from "vitest";
+   import { connectRedis, deleteValue, disconnectRedis, getValue, setValue } from ".";
+   import { RedisContainer, StartedRedisContainer } from "@testcontainers/redis";
+   import { Wait } from "testcontainers";
+   import { createClient } from "redis";
+
+   let container: StartedRedisContainer;
+
+   beforeAll(async () => {
+       container = await new RedisContainer()
+           .withExposedPorts(6379)
+           .withWaitStrategy(Wait.forLogMessage("Ready to accept connections"))
+           .start();
+
+       await connectRedis(`redis://localhost:${container.getMappedPort(6379)}`);
+   });
+
+   afterAll(async () => {
+       await disconnectRedis();
+   });
+
+   beforeEach(async () => {
+       // Flushind DB and adding to Redis some values before each test
+       const client = createClient({ url:`redis://localhost:${container.getMappedPort(6379)}` });
+       await client.connect();
+
+       await client.flushDb();
+       await client.set('preset-key', 'preset-value');
+       await client.set('preset-key1', 'preset-value1');
+       await client.quit();
+   });
+
+   test("set value on server", async () => {
+       // Set value
+       const ret = await setValue("key", "value");
+       expect(ret).toBe("OK");
+
+       // Update value
+       const ret1 = await setValue("key", "updated-value");
+       expect(ret1).toBe("OK");
+   });
+
+   test("get value from server", async () => {
+       // Get preset value
+       const value = await getValue("preset-key");
+       expect(value).toBe("preset-value");
+
+       // Get not existing value
+       const value1 = await getValue("key");
+       expect(value1).toBeNull();
+   });
+
+   test("delete value on server", async () => {
+       // Delete two records in a same time
+       const res = await deleteValue(["preset-key", "preset-key1"]);
+       expect(res).toBe(2);
+
+       // Delete not existing record
+       const res1 = await deleteValue(["key"]);
+       expect(res1).toBe(0);
+   });
+   ```
+
+## Running tests
+
+When running Testcontainers for the first time, ensure that you run your tests in `DEBUG` mode by using this command:
+
+```shell-session
+$ DEBUG=testcontainers* npm test
+```
+
+Then, you should be able to see lines similar to the ones below:
+
+```console
+testcontainers [DEBUG] Loading ".testcontainers.properties" file...
+testcontainers [DEBUG] Loaded ".testcontainers.properties" file
+testcontainers [DEBUG] Found custom configuration: dockerHost: "unix:///run/user/1000//podman/podman.sock
+```
+
+Those lines indicate that Testcontainers found the configuration file, and the containers are using the Podman engine instead of Docker.
```

Comment:
```suggestion
Those lines indicate that Testcontainers found the configuration file, and the containers are created on the Podman engine instead of Docker.
```